src/content/drafts/introducing-bulk-suppressions.md

```diff
@@ -0,0 +1,94 @@
+categories:
+  - Announcements
+---
+
+At some point, every development team faces the same challenge: you want to enable a new ESLint rule to improve code quality, but your existing codebase already contains numerous violations. When the rule isn't auto-fixable, the path to enabling it as an error can be challenging.
+
+Today, we're excited to introduce ESLint's new violation suppression system, designed to make adopting stricter linting rules more manageable. Let's explore how it works and how it can help your team enhance code quality incrementally.
+
+## Why bulk suppressions?
+
+When enabling a rule in an established project, teams typically choose between fixing all violations upfront, using warnings instead of errors, disabling rules in specific files, or adding inline disable comments. Each approach has merit, but none provides an ideal balance between enforcing standards for new code while addressing existing violations over time.
+
+ESLint's new suppression system offers a more flexible approach by:
+
+1. Recording existing violations in a separate file
+2. Enforcing the rule as an error for all new code
+3. Suppressing recorded violations, allowing you to address existing violations at your own pace
+
+This approach prioritizes preventing new violations while providing a clear path to incrementally improve existing code.
+
+## Getting started
+
+Once you've configured a new rule as `"error"` in your ESLint configuration, you can bulk suppress existing violations using the command line. You can choose to suppress all violations or specific rules.
+
+```
+# Suppress all violations
+eslint --suppress-all
+
+# Suppress specific rules
+eslint --suppress-rule <rule-name>
+```
+
+When you execute any of the above commands, ESLint will create a file called `eslint-suppressions.json` in the project root. This file contains the number of violations for each rule and file. The suppressions file is structured as follows:
+
+```json
+{
+  "src/file1.js": {
+    "no-undef": {
+        "count": 1
+    }
+  },
+  "src/file2.js": {
+    "no-unused-expressions": {
+        "count": 2
+    }
+  }
+}
+```
+
+ESLint will suppress all `"error"` violations for the specified rules in subsequent runs and will not treat them as errors. However, if more violations are found for the same rule in the same file, ESLint will still report **all** of them in the output. For example, if a file originally had 2 suppressed errors but now has 5, ESLint will display all 5 errors.
+
+This approach is intentional and helps avoid confusion. Since suppressions are tracked per file and rule—not per specific line or code change—there’s no reliable way to determine whether the new violations were introduced recently or already existed. Rather than trying to guess which violations to hide, ESLint chooses to show the full picture.
+
+Please note that it's recommended to combine the above commands with the `--fix` flag to fix any violations that can be automatically resolved.
```

Comment:
I think we should move this note right under the actual commands. It's a bit out of place here.